src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -238,6 +239,20 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                     throw new \Exception('Unable to clone code repository: ' . $stderr);
                 }
 
+                // Local refactoring for function folder with spaces
+                if (str_contains($rootDirectory, ' ')) {
+                    $rootDirectoryWithoutSpaces = str_replace(' ', '', $rootDirectory);
+                    $from = $tmpDirectory . '/' . $rootDirectory;
+                    $to = $tmpDirectory . '/' . $rootDirectoryWithoutSpaces;
+                    $exit = Console::execute('mv "' . $from . '" "' . $to . '"', '', $stdout, $stderr);
```

Comment:
we should probably use `escapeshellarg` to clean up the commands before we run them